PREREQUISITES.md

```diff
@@ -51,14 +51,42 @@ openssl genrsa -3 -out private_rsa_key.pem 3072
 export PDO_ENCLAVE_PEM=`pwd`/private_rsa_key.pem
 ```
 
+# <a name="packages"></a>Required Packages
+On a minimal Ubuntu system, the following packages are required. Other
+distributions will require similar packages.
+```
+sudo apt-get update
+sudo apt-get install -y cmake swig pkg-config python3-dev python3-venv software-properties-common python3-dev virtualenv curl tinyscheme git unzip
+```
+
+# <a name="protobuf"></a>Protobuf Compiler
+Many components of the project use Google's Protocol Buffers (including SGX),
+so installing support for them early is recommended. Protobuf v3 or later
+support is required - check your package manager first to see what is
+available. If a package is not available, follow these steps to compile and
+install protobuf tools manually:
+
+```
+wget https://github.com/google/protobuf/releases/download/v3.5.1/protobuf-python-3.5.1.tar.gz
+tar xzf protobuf-python-3.5.1.tar.gz
+cd protobuf-3.5.1
+./configure
+make -j16
+make check -j16
+sudo make install
+export LD_LIBRARY_PATH=/usr/local/lib
+```
+
 # <a name="sgx"></a>Software Guard Extensions (SGX)
 Hyperledger Private Data Objects is intended to be run on SGX-enabled
 Intel&reg; platforms. However, it can also be run in "simulator mode" on
 platforms that do not have hardware support for SGX.
 
 If you plan to run this on SGX-enabled hardware, you will need the SGX driver,
-PSW, and SDK. If running only in simulator mode (insecure!), you only need the
-SGX SDK.
+PSW, and SDK. If running only in simulator mode (no hardware support), you only
+need the SGX SDK. To learn more about Intel SGX, read the Intel SGX SDK
+doumentation [here](https://software.intel.com/en-us/sgx-sdk/documentation) or
+visit the Intel SGX homepage [here](https://software.intel.com/en-us/sgx).
```

Comment:
We need to note possible situation when an older version SGX SDK can be picked if not overwritten, something like
"It is recommended to install Intel SGX SDK in /opt/intel/sgxsdk because later you may need to build SGX OpenSSL library that expects Intel SGX SDK in this location unless you change SGX OpenSSL build script"

Reply:
Good point! Amended into the last commit.